src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -1065,8 +1067,9 @@ def __import_kvset_from_file(client, path, yes):
         elif not validate_import_key(config_setting.key):
             continue
```

Comment:
This part is validating the key. Can we move those checks into __validate_import_config_setting, where label, value, content_type are validated.  Also it should be done before printing preview.

Reply:
Moved the logic to the `__validate_import_config_setting` and it now validates before print preview.